x-pack/plugins/threat_intelligence/cypress/e2e/indicators.cy.ts

```diff
@@ -45,12 +45,39 @@ const THREAT_INTELLIGENCE = '/app/security/threat_intelligence/indicators';
 const URL_WITH_CONTRADICTORY_FILTERS =
   '/app/security/threat_intelligence/indicators?indicators=(filterQuery:(language:kuery,query:%27%27),filters:!((%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:file),type:phrase),query:(match_phrase:(threat.indicator.type:file))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:url),type:phrase),query:(match_phrase:(threat.indicator.type:url)))),timeRange:(from:now/d,to:now/d))';
 
+Cypress.Cy.prototype.onUncaughtException = function () {};
```

Comment:
TODO: remove this before merging, this is to allow testing with Cypress, without it Cypress crashes before the tests runs because of an unhandled exception